features/config.feature

```diff
@@ -92,6 +92,43 @@ Feature: Config
       The requested config file does not exist
       """
 
+  Scenario: PHP configuration file
+    Given a file named "behat.php" with:
+      """
+      <?php
+
+      return new Behat\Config\Config();
+
+      """
+    And a file named "features/bootstrap/FeatureContext.php" with:
+      """
+      <?php
+
+      use Behat\Behat\Context\Context;
+
+      class FeatureContext implements Context
+      {
+      }
+      """
+    And a file named "features/config.feature" with:
+      """
+      Feature:
+        Scenario:
+          When this scenario executes
+      """
+    When I run "behat -f progress --no-colors --append-snippets --config=behat.php"
+    Then it should pass with:
+      """
+      U
+
+      1 scenario (1 undefined)
+      1 step (1 undefined)
+
+      --- Use --snippets-for CLI option to generate snippets for following default suite steps:
+
+          When this scenario executes
+      """
+
```

Comment:
I would probably add a test to confirm that the `--config` command line option lets you pick a config file different from the default behat.php

Reply:
yes, good idea

Reply:
I've added one scenario with alternative PHP configuration file.